src/client/mediawiki/MwWindow.ts

```diff
@@ -7,10 +7,17 @@ interface MwConfig {
 
 type MwTitle = new( title: string, namespace: number ) => any;
 
+export interface MwMessageText {
```

Comment:
I find this super confusing, but it's mediawiki's fault. To me, this is the thing that should be called `Message`, and the one below is the service/function that spits out message objects. Naming is hard :[

Reply:
done